File: introduction/ITP1_3_D-How Many Divisors?.cs
using System;

public class Program {
  public static void Main() {
    var line = Console.ReadLine().Split();

    int a = int.Parse(line[0]);
    int b = int.Parse(line[1]);
    int c = int.Parse(line[2]);
    int count = 0;
    while(a <= b) {
      if(c % a == 0) {
        count++;
      }
      a++;
    }
    
    Console.WriteLine(count);
  }
}

File: introduction/ITP1_3_D-How Many Divisors?.kt
fun main(args:Array<String>) {
  val line = readLine()!!.split(" ").map{it.toInt()}
  var n = 0
  for (i in line[0]..line[1]) {
    if (line[2] % i == 0) n++
  }
  println(n)
}
